charts/merlin/templates/_helpers.tpl

```diff
@@ -57,3 +57,12 @@
 {{- printf .Values.mlflow.backendStoreUri -}}
 {{- end -}}
 {{- end -}}
+
+
+{{- define "merlin.getEnvironmentSecret" -}}
```

Comment:
I prefer to rename it to `merlin.environmentsSecretName` because it only returns a string for the secret name instead of the whole environments object.